src/PlayTurn.java

```diff
@@ -1,75 +1,94 @@
+import java.io.*;
 import java.util.Scanner;
 
 public class PlayTurn {
-    //setting the object for the winCondition class 
-    WinCondition winCondition=new WinCondition();
-    MainGame mainGame=new MainGame();
-    
-  //a method to confirm the player symbol
-    public static void settingPlayerSymbol(int playerNo,char playerSymble)
-    {
+    // setting the object for the winCondition class
+    MainGame mainGame = new MainGame();
+
+   
+    /**
+     * a method to confirm the player symbol
+     * @param playerNo : the number of the player that is playing
+     * @param playerSymble : the Player Symbol
+     */
+    public static void settingPlayerSymbol(int playerNo, char playerSymble) {
         Scanner scan = new Scanner(System.in);
-        System.out.println("Player "+playerNo+" YOUR SYMBOL IS:"+playerSymble+"\nIs that Correct?");
-        char confirm=scan.next().charAt(0);
-        //Confirming player Symbol
-        if(confirm=='n'||confirm=='N')
-        {
-            System.out.println("Player "+playerNo+" ENTER YOUR NEW SYMBOL:");
+        System.out.println("Player " + playerNo + " YOUR SYMBOL IS:" + playerSymble + "\nIs that Correct?");
+        char confirm = scan.next().charAt(0);
+        // Confirming player Symbol
+        if (confirm == 'n' || confirm == 'N') {
+            System.out.println("Player " + playerNo + " ENTER YOUR NEW SYMBOL:");
             playerSymble = scan.next().charAt(0);
         }
     }
-    
-    
-    //a public method to get the input of the player and setting it
-    public void playerTurn(char[][] block,char playerSymble,boolean gameWon,int turn,int playernumber) {
 
-        System.out.println("Player "+playernumber+" Enter Your Move:");
+ 
+    /**
+     * a public method to get the input of the player and setting it
+     * @param block : the game board
+     * @param playerSymble : the symbol of the current player
+     * @param gameWon : check if the game won or not
+     * @param turn :the number of turns
+     * @param playernumber :the number of the playing player
+     */
+    public static void playerTurn(char[][] block, char playerSymble, boolean gameWon, int turn, int playernumber) {
+
+        System.out.println("Player " + playernumber + " Enter Your Move:");
         Scanner scan = new Scanner(System.in);
         char input = scan.next().charAt(0);
         boolean inputFound = false;
-        
-        //do while loop will repeat until the player input an available/valid position
```

Comment:
repeat the same comment, remove one

Reply:
red mean the old file not the pull requested file there is no repeated lines in the code